core/src/main/java/org/axonframework/saga/repository/jpa/AssociationValueEntry.java

```diff
@@ -33,7 +34,7 @@
 public class AssociationValueEntry {
 
     @Id
-    @GeneratedValue
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
CIM doesn't use sagas as far as I know. We need to make 100% sure of that. Add it to the readme.